### DISCUSSION TITLE
Opacity modifier to work with text-opacity-{x}

### DISCUSSION THREAD

**Discussion by mason-rogers** [tailwindlabs/tailwindcss#4717] [Ideas]
I have plenty of cases where I currently have `text-white text-opacity-50 hover:text-opacity-80`, I imagined when using the opacity modifiers I would be able to keep the hover class the same and not have to completely overwrite the color on hover using another opacity shortcut class.

With these modifiers I now have to do `text-white/50 hover:text-white/80` because the opacity modifiers don't use a variable for opacity meaning it cant be overwritten by different states. I think that it would be nice if I didn't have to worry about changing 2 colors if I ever changed anything for the best DX, by allowing `text-white/50 hover:text-opacity-80`.

**Comment by JakovGlavac** [comment]
Try `text-opacity-[0.5]` or `text-opacity-[50%]` with JIT enabled

**Reply by mason-rogers** [reply]
![image](https://user-images.githubusercontent.com/26467584/122652455-dd874100-d136-11eb-91f5-20bc85252c95.png)
Doesn't work, same issue as it still uses a variable for the arbitrary value.
